src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationDefaults.cs

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.EVEOnline {
```

Comment:
I suppose you're not a huge fan of the alternate `EveOnline` casing? It seems better, tho.